modules/quant_api/src/Form/SettingsForm.php

```diff
@@ -56,18 +56,18 @@ public function buildForm(array $form, FormStateInterface $form_state) {
 
     if ($config->get('api_token')) {
       if ($project = $this->client->ping()) {
-        $message = t('Successfully connected to @api', ['@api' => $config->get('api_project')]);
+        $message = $this->t('QuantAPI status: Successfully connected to project <code>@project</code>', ['@project' => $config->get('api_project')]);
         \Drupal::messenger()->addMessage($message);
       }
       else {
-        \Drupal::messenger()->addError(t('Unable to connect to Quant API, check settings.'));
+        \Drupal::messenger()->addError($this->t('QuantAPI error: Unable to connect to the Quant API please check the endpoint on the <code>Integrations</code> page in the Quant dashboard.'));
       }
     }
 
     $form['api_endpoint'] = [
       '#type' => 'textfield',
       '#title' => $this->t('API Endpoint'),
-      '#description' => $this->t('e.g: https://api.quantcdn.io'),
+      '#description' => $this->t('The fully-qualified domain name for the API endpoint, e.g. <code>https://api.quantcdn.io</code>'),
```

Comment:
I think we should update this one so that the API endpoint is hardcoded and requires drush access to change 9/10 use cases will never need to change this value.